app-benchmarks/cpuburn/cpuburn-1.4a-r2.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
```

Comment:
why?

Reply:
Sorry, forgot to delete after switch to newer EAPI. Saw it today early, but forgot to says here, that I will fix it. Done now.